docker/Dockerfile-i686

```diff
@@ -6,7 +6,9 @@ ENV AUDITWHEEL_PLAT manylinux2014_$AUDITWHEEL_ARCH
 ENV LC_ALL en_US.UTF-8
 ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US.UTF-8
-ENV LD_LIBRARY_PATH /usr/local/lib64:/usr/local/lib
+ENV DEVTOOLSET_ROOTPATH /opt/rh/devtoolset-9/root
+ENV PATH $DEVTOOLSET_ROOTPATH/usr/bin:$PATH
+ENV LD_LIBRARY_PATH $DEVTOOLSET_ROOTPATH/usr/lib:$DEVTOOLSET_ROOTPATH/usr/lib:$DEVTOOLSET_ROOTPATH/usr/lib/dyninst:/usr/local/lib
```

Comment:
`$DEVTOOLSET_ROOTPATH/usr/lib` is duplicated

Reply:
thanks, should be fixed now.

Reply:
Also fixed in manylinux2010 PR (the mistake comes from there originally, copy/paste...)